lib/ansible/modules/system/mdadm.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2015, Adam Hamsik <haaaad@gmail.com>
+#
+# This file is part of Ansible
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: mdadm
+version_added: 2.3
+short_description: Initialize and manage linux mdadm array
+description: Create, Manage linux software raid
+requirements: [ python json ]
+options:
+  state:
+    description:
+      - Defines action which can be either RAID creation or removal. This will assemble
+        existing array if it exist od disks or it will try to create new one if it doesn't.
+    choices: [ 'present', 'absent', 'examine', 'scan']
+    default: 'present'
+    required: false
+  level:
+    description:
+      - type of array to create
+    required: true
+    default: 1
+    choices: ['0', '1', '5', '6']
```

Comment:
Here, the choices seem to include 10.
Is it intended?

https://github.com/ansible/ansible/pull/20275/files#diff-61154e3cce92f549c3b97827d6311550R311